src/qz/utils/ShellUtilities.java

```diff
@@ -240,6 +240,10 @@ public static String executeRaw(String[] commandArray) {
                     }
```

Comment:
If its going to check for a missing value after the loop, the `NumberFormatException` block should be emptied to avoid the duplicate code.

Reply:
@MrL0co once you address @bberenz's comment, we'd happily merge.

Reply:
Yes, Sorry i didn't have time during work to address this due to deadlines. It is fixed now.